src/test/java/com/fruits/ecommerce/integration/AuthControllerIntegrationTest.java

```diff
@@ -0,0 +1,224 @@
+package com.fruits.ecommerce.integration;
+
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fruits.ecommerce.Models.DTOs.LoginRequestDTO;
+import com.fruits.ecommerce.Models.DTOs.UserDTO;
+import com.fruits.ecommerce.Models.Entities.Role;
+import com.fruits.ecommerce.Models.Enum.RoleType;
+import com.fruits.ecommerce.Repository.RoleRepository;
+import com.fruits.ecommerce.Repository.UserRepository;
+import org.junit.jupiter.api.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.*;
+import org.springframework.security.test.context.support.WithMockUser;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.TestExecutionListeners;
+import org.springframework.test.context.support.DependencyInjectionTestExecutionListener;
+import org.springframework.test.context.support.DirtiesContextTestExecutionListener;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.delete;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+//@SpringBootTest
```

Comment:
Let's remove this commented block here

Reply:
Well done, I will work on implementing it for the better as you indicated.